content/en/security/application_security/api-inventory/_index.md

```diff
@@ -45,49 +65,67 @@ To use API Security on your services, **you must have AAP Threats Protection ena
 
 **Note**: On .NET Core and .NET Fx tracers, you need to set the environment variable `DD_API_SECURITY_ENABLED=true` for API Security features to work properly.
```

Comment:
We need to add proxies here as well, @Julio-Guerra do we have any docs already we could leverage?

Reply:
I think I'll just like to the setup docs here: https://docs.datadoghq.com/security/application_security/setup/
That way they can see what's needed for their APIs and services